broker.go

```diff
@@ -84,50 +92,44 @@ func handleDefaultPort(w http.ResponseWriter, r *http.Request) {
 }
 
 func handleMessage(s *discordgo.Session, m *discordgo.MessageCreate) {
-	// Ignore all messages created by the bot itself
-	// This isn't required in this specific example but it's a good practice.
+	/* Validation */
 	if m.Author.ID == s.State.User.ID {
-		log.Println("Ignore the bot's message itself")
 		return
 	}
-
-	trimmed := strings.TrimSpace(m.Content)
-
-	// If not in the format "!stonks <ticker>", give up.
-	if !strings.HasPrefix(trimmed, "!stonks ") {
-		log.Println("Ignoring non-prefixed message")
+	if !strings.HasPrefix(strings.TrimSpace(m.Content), "!stonks ") {
 		return
 	}
 
 	ticker := strings.TrimPrefix(m.Content, "!stonks ")
 
 	if ticker == "" {
+		// TODO: Send a help message to the user.
 		log.Println("Empty stock ticker")
 		return
 	}
 
+	/* Serving */
+
 	value, err := findTicker(ticker)
 	if err != nil {
-		s.ChannelMessageSend(m.ChannelID, fmt.Sprintf("Error with Ticker: %s", ticker))
-		log.Fatalln("error fetching ticker:", ticker, err)
+		msg := fmt.Sprintf("failed to get quote for ticker %q: %v", ticker, err)
```

Comment:
Why should we return the error to the user? I think we should log it, but the long form error message should only be logged server side

Reply:
I figured we're technical enough to want the errors in discord. Up to you.